src/renderer/utils/scrollToIssue.js

```diff
@@ -0,0 +1,9 @@
+export default (issue, opts = {}) => {
```

Comment:
```suggestion
export default (issue, opts = { behavior: 'smooth' }) => {
```
ja zrobiłbym tak, a w componencie tylko podawać dokąd skrollować,  w 99% przypadków optsy nie zmienią się

+ fajnie by było mieć możliwość użyć ten skrypcik w innych miejscach
(zmienić nazwe na scrollTo, w argumentach mieć (el, opts), a w componenncie podać document.getElementById(issue.key)